frontend/src/components/Dashboard/ScoresTableSection/BodyFields.jsx

```diff
@@ -1,37 +0,0 @@
-import React from 'react'
-import {connect} from 'react-redux'
-import {TableBody, TableRow, TableRowColumn} from 'material-ui'
-
-
-class BodyFields extends React.Component {
-  render() {
-    const subjectsList = this.props.groupStore.subjects.list
```

Comment:
This file is currently excluded from the following releases.
(Stop counting, I've got you).